sd_jwt/bin/sd_jwt

```diff
@@ -0,0 +1,252 @@
+    '--indent', required=False,
+    type=int,
+    default=4,
+    help=("json output indentation level")
+)
+parser.add_argument(
+    '--replace-examples-in', required=False,
+    type=str,
+    help=(
+        "path/to/main.md"
+        " - replaces all the examples in the specs"
+    )
+)
+parser.add_argument(
+    '-v', '--version', required=False,
+    action="store_true",
+    help="Print version and exit"
+)
+
+_args = parser.parse_args()
+logger.setLevel(_args.debug)
+
+if _args.version:
+    sys.exit(f'{__version__}')
+
+# load settings
+if _args.settings_path:
+    sys.path.append(_args.settings_path)
+    from settings import *
+else:
+    from sd_jwt.demo_settings import *
+
+if _args.example:
+
+    print("# Creating the SD-JWT")
+    
+    # They may came from a custom python settings file :-)
+    jwk_kwargs = {
+        "iss_key": ISSUER_KEY_DICT,
+        "holder_key": HOLDER_KEY_DICT,
+        "key_size": DEFAULT_KEY_SIZE,
+        "kty": DEFAULT_KTY
+    }
+    _JWKS = get_jwk(jwk_kwargs, _args.no_randomness)
+    ISSUER_KEY = _JWKS["ISSUER_KEY"]
+    HOLDER_KEY = _JWKS["HOLDER_KEY"]
+    ISSUER_PUBLIC_KEY = _JWKS["ISSUER_PUBLIC_KEY"]
+    #
+    
+    if _args.example == 'simple':
+        SD_JWT_SCG_ARGS = (SIMPLE_USER_CLAIMS, ISSUER, ISSUER_KEY, HOLDER_KEY)
+        DISCLOSED_CLAIMS = SIMPLE_DISCLOSED_CLAIMS
+    
+    elif _args.example == 'structured':
+        SD_JWT_SCG_ARGS = (
+            SIMPLE_USER_CLAIMS, ISSUER, ISSUER_KEY, HOLDER_KEY, STRUCTURED_CLAIMS_STRUCTURE
+        )
+        DISCLOSED_CLAIMS = STRUCTURED_DISCLOSED_CLAIMS
+    
+    elif _args.example == 'complex':
+        SD_JWT_SCG_ARGS = (
+            COMPLEX_USER_CLAIMS, ISSUER, ISSUER_KEY, HOLDER_KEY, COMPLEX_CLAIMS_STRUCTURE
+        )
+        DISCLOSED_CLAIMS = COMPLEX_DISCLOSED_CLAIMS
+    
+    else:
+        raise NotImplementedError(_args.type)
+    
+    
+    print_repr(
+        ("User claims", json.dumps(SD_JWT_SCG_ARGS[0], indent=_args.indent))
+    )
+    
+    sd_jwt_payload, serialized_sd_jwt, svc_payload, serialized_svc = create_sd_jwt_and_svc(
+        *SD_JWT_SCG_ARGS, iat=_args.iat, exp=_args.exp
+    )
+    
+    print_repr(("Payload of the SD-JWT:",
+               json.dumps(sd_jwt_payload, indent=_args.indent)))
+    print_repr(("The serialized SD-JWT:", serialized_sd_jwt))
+    print_repr(("Payload of the SD-JWT Salt/Value Container (SVC):",
+               json.dumps(svc_payload, indent=_args.indent)))
+    print_repr(("The serialized SD-JWT Salt/Value Container (SVC):", serialized_svc))
+    combined_sd_jwt_svc = f"{serialized_sd_jwt}{serialized_svc}"
+    
+    print("Serialized SD-JWT Salt/Value Container (SVC) decoded representation:")
+    print_decoded_repr(serialized_sd_jwt)
+    
+    #####
+    print("# Creating the SD-JWT-Release")
+    sd_jwt_release_payload, serialized_sd_jwt_release = create_release_jwt(
+        _args.nonce, ISSUER, DISCLOSED_CLAIMS, serialized_svc, HOLDER_KEY
+    )
+    
```

Comment:
Isn't the audience verifier?
```suggestion
    #####
    print("# Creating the SD-JWT-Release")
    sd_jwt_release_payload, serialized_sd_jwt_release = create_release_jwt(
        _args.nonce, VERIFIER, DISCLOSED_CLAIMS, serialized_svc, HOLDER_KEY
    )
    
```

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/6a7dbc6bbec4ad807ad2b1f1107375a66d3cfecf